ChurchToolsApi.py

```diff
@@ -564,19 +564,22 @@ def file_download(self, filename: str, domain_type, domain_identifier, path_for_
             arrangement_files = response_content['data'].copy()
             logging.debug("SongArrangement-Files load successful {}".format(response_content))
             fileUrl = None
+            file_found = False
 
             for file in arrangement_files:
                 filenameoriginal = str(file['name'])
                 fileUrl = str(file['fileUrl'])
```

Comment:
this part is executed against each file but is only required with the downloadfile. Wouldn't it be better to move it between 579 and 580 just before it is used and only in case a match is found?